### PR TITLE
Test for the redirect when the file checks are complete

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: src/test/resources/features/FileChecks.feature [hunk 1]
```diff
@@ -19,7 +19,7 @@ Feature: File Checks Page
     Then the user will be on a page with the title "Checking your records"
     And the checksum progress bar should be visible
     And the checksum progress bar should have 25% progress
-
+v
```
Comment (TomJKing) [inline src/test/resources/features/FileChecks.feature:22]: Is the "v" necessary?

File: src/test/resources/features/FileChecks.feature [hunk 2]
```diff
@@ -28,4 +28,12 @@ Feature: File Checks Page
...
     And the user clicks the continue button
+    Then the user will be on a page with the title "Record check results"
+
+  Scenario: User is redirected to the results page if the checks are complete and they visit the record checks page
+    Given A logged out user
+    And an existing consignment for transferring body MOCK1
+    And an existing transfer agreement
+    And the records checks are complete
+    When the user is logged in on the records page
     Then the user will be on a page with the title "Record check results"
```
Comment (TomJKing) [inline src/test/resources/features/FileChecks.feature:39]: Missing line break at end of file

**Review by TomJKing — COMMENTED** [review]
Couple of minor comments.

File: src/test/resources/features/FileChecks.feature (hunk 1 above)
Comment (MancunianSam) [reply src/test/resources/features/FileChecks.feature:22]: :smile: Poor keyboard control strikes again.

File: src/test/resources/features/FileChecks.feature (hunk 2 above)
Comment (MancunianSam) [reply src/test/resources/features/FileChecks.feature]: I've added one in.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**